tox_packet/src/messenger/conference/invite.rs

```diff
@@ -73,6 +73,6 @@ mod tests {
     encode_decode_test!(
         tox_crypto::crypto_init().unwrap(),
         invite_encode_decode,
-        Invite::new(1, ConferenceType::Text, ConferenceUid::random())
+        Invite::new(1, ConferenceType::Text, ConferenceUid([42; CONFERENCE_UID_BYTES]))
```

Comment:
```suggestion
        Invite::new(1, ConferenceType::Text, ConferenceUid([42; ConferenceUid::BYTES]))
```
?

Reply:
We can change it to this format. but then we need to review other consts as well. And I wouldn't call it `BYTES`, but `SIZE` or similar.